README.md

```diff
@@ -63,3 +63,15 @@ Press CTRL+C to quit
  * Debugger PIN: 113-666-066
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing the App
+
+Tests are run using pytest, you can install it by running:
+```bash
+$ poetry install pytest
```

Comment:
It always fights instinct if you're used to e.g. npm, but actually!
```suggestion
$ poetry add pytest
```

(And I'm always torn on whether this is worth documenting - helpful for the sake of this course certainly, but future devs won't need to run this because `poetry install` will cover it!)